server/src/testflinger/database.py

```diff
@@ -388,3 +399,24 @@ def get_agents() -> list[dict]:
     """Return a list of all agents."""
     agents = mongo.db.agents.find({}, {"_id": False, "log": False})
     return list(agents)
+
+
+def add_restricted_queue(queue: str, client_id: str):
+    """Add a restricted queue for a client."""
```

Comment:
Could you please add a description for params:
```suggestion
    """Add restriction for a specified queue. 
    
    :param queue: Queue name to add to restricted_queue collection
    """
```